install_with_docker.sh

```diff
@@ -321,11 +325,9 @@ if [ -n "${CONFIG_FILE}" ]; then
     CFILE=${CONFIG_FILE}
 fi
 
-
-
-# Run Docker compose files
-UI_PORT=${UI_PORT} CONFIG_FILE=${CFILE} WORK_DIR="$(pwd)" SIGLENS_VERSION=${SIGLENS_VERSION} docker-compose -f ./docker-compose.yml up -d || {
-    post_event "install_failed" "Failed to start Docker Compose on $os with docker-compose.yml"
+print_success_message "\n Starting Siglens with image: ${DOCKER_IMAGE_NAME}"
+UI_PORT=${UI_PORT} CONFIG_FILE=${CFILE} WORK_DIR="$(pwd)" SIGLENS_VERSION=${SIGLENS_VERSION} IMAGE_NAME=${DOCKER_IMAGE_NAME} docker-compose -f $DOCKER_COMPOSE_FILE up -d || {
```

Comment:
you probably don't need to pass the siglens_version env variable to docker-compose, now that you have removed it from the compose yaml file ?